makefile.shade

```diff
@@ -33,8 +38,26 @@ k-standard-goals
       "-n ResponseCachingPlaintextRequestNoCache",
       "-n ResponseCachingPlaintextVaryByCached",
     };
+
+    // Expect failures with HttpSys if BenchmarksServer is running on Linux. Never expect two failures in a row.
```

Comment:
Ignoring failures might cause us to miss real issues.  Another option would be to add a new `ServerState` named something like `NotSupported`, then update `BenchmarksServer` and `BenchmarksDriver` to flow the `NotSupported` state rather than `Failed`.